spec/rubocop/cop/style/unneeded_percent_q_spec.rb

```diff
@@ -34,24 +34,22 @@
       expect(cop.messages).to be_empty
     end
 
-    it 'registers an offense for a string with single quotes and what appears' \
-       ' to be an escape' do
-      # There is no such thing as escapes in a %q() string
-      # So this can just as well be written with double quotes
-      inspect_source(cop, "%q('hi\\t')")
```

Comment:
This test was not correct. The source code string evaluates to `%q('hi\t')`, i.e. there is an escaped `t`. Replacing `%q` here gives a different string:
```
irb(main):001:0> puts "%q('hi\\t')"
%q('hi\t')
=> nil
irb(main):002:0> puts %q('hi\t')
'hi\t'
=> nil
irb(main):003:0> puts "'hi\t'"
'hi     '
```